salt/cloud/deploy/bootstrap-salt.sh

```diff
@@ -2530,6 +2530,7 @@ install_debian_8_git_deps() {
         __REQUIRED_TORNADO="$(grep tornado "${__SALT_GIT_CHECKOUT_DIR}/requirements/base.txt")"
         if [ "${__REQUIRED_TORNADO}" != "" ]; then
             __apt_get_install_noinput python-tornado
+            __apt_get_install_noinput python-pycurl
```

Comment:
@mrproper, bootstrap changes should be made at https://github.com/saltstack/salt-bootstrap.  Salt bootstrap development happens somewhat independently from salt and bootstrap-salt.sh is regularly merged into the salt codebase.